src/components/table/sort.ts

```diff
@@ -9,11 +9,20 @@ export function descendingComparator<T>(a: T, b: T, orderBy: keyof T): number {
   const bValue = b[orderBy] ?? '';
   const aValue = a[orderBy] ?? '';
 
-  if (bValue < aValue) {
+  // blank values at the end always (any order)
+  if (aValue === '' || aValue === null) {
```

Comment:
suggestion use trim:
```
  if (aValue === null || aValue === undefined || aValue.trim() === '') {
  }
```

Reply:
just fyi - `trim` will only work if it is a string, maybe aValue.toString().trim()?